terraform/aks/database.tf

```diff
@@ -1,5 +1,6 @@
 module "postgres" {
-  source = "./vendor/modules/aks//aks/postgres"
+  count                        = startswith(var.environment, "review") ? 0 : 1
```

Comment:
As a best practice, we don't hardcode environment names in the code. Instead, we normally define a boolean variable such as "deploy_database" and set it to true or false in the environment variables. 
So this would become:
```
count = var.deploy_database ? 1 : 0
```